sata.go

```diff
@@ -668,6 +671,20 @@ func checkTempRange(t int8, t1 int8, t2 int8, lo *int8, hi *int8) bool {
 	return false
 }
 
+func (a AtaSmartAttr) AttributeFlagsPrefailure() string {
```

Comment:
These methods add human representation for flags. But I do not see how useful it will be for a smart.go user.

smart.go is a library and `a.Flags` is intended to be used directly. Converting this to a human-readable string just to make a string comparison adds unnecessary overhead.

These `AttributeFlagsPrefailure` and `AttributeFlagsOnline` methods need to be removed. And instead, add more information to the Flags documentation.

Reply:
Ok, I got the idea.
But what about simplification of the `Flags` analysis? What if the functions added will be remained, but not with `string` output, but `bool`? E.g.:
```suggestion
func (a AtaSmartAttr) AttributeFlagsPrefailure() bool {
	if a.Flags&ATTRIBUTE_FLAGS_PREFAILURE != 0 {
		return true
	}
	return false
}
```